bookmark.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BOOKMARK_H
+#define BOOKMARK_H
+#include <string>
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+class Bookmark {
+        public:
+            void displayMainMenu();
+            void addUser(int userCounter);
+            void editUser(int userCounter);
+            void viewAllUsers(int userCounter);
+            void deleteUser(int userCounter);
+
+            void addBook(int bookCounter);
+            void deleteBook(int bookCounter);
+            void editBook(int bookCounter);
+            void searchBook(int bookCounter);
+            void viewAllBooks(int bookCounter);
+            void displayTopBooks();
+
+
+//counter for book array and user array
+    int bookCounter = 0;
+    int userCounter=0;
+
+//function to increase counter
+    void increment(int a) {
+        bookCounter = ++a;
+    }
+
+    void increment1(int b) {
```

Comment:
what is the difference between increment and increment1?

Reply:
"increment" increments the counter for the "books" array, and "increment1" increments the counter for the "users" array. More precisely, one increment increases the number of users, and the second increment increases the number of books.